python/py-pysam/Portfile

```diff
@@ -5,12 +5,11 @@ PortGroup           python 1.0
 
 name                py-pysam
 set realname        pysam
-version             0.12.0.1
-revision            1
+version             0.15.1
 categories-append   science
 platforms           darwin
 license             MIT BSD
-maintainers         @SoapZA
+maintainers         users.noreply.github.com:SoapZA openmaintainer
```

Comment:
See my comment on the other pull request (which is still open and conflicts with this one it seems?)

Reply:
As @pmetzger said, ideally the line should look like this:
```
maintainers         {users.noreply.github.com:SoapZA @SoapZA} openmaintainer
```
but with a valid obfuscated email. I strongly suspect that `users.noreply.github.com` is what it says it is: an email you cannot really write to. Having an invalid email is not any more valid or helpful than not having any email listed at all.

While it might theoretically be feasible to avoid specifying a valid email, there are parts of our architecture that still need it.
* You get an email if someone else makes a change in the port (this might be necessary if a library dependency changes in an ABI-incompatible way and the port needs a revbump/rebuild etc.) and the port fails to build on our buildbot.
* When users submit a bug report to Trac, Trac needs to send you an email notification and neither users nor Trac can notify you about bugs in the absence of a valid email. That said, you need to register with some email on Trac anyway (unless you never register at all); Trac also sees the emails you listed on your GitHub account, I think it picks up one if you don't specify another one yourself.
(There are a few other less important instances.)